go.mod

```diff
@@ -66,7 +66,7 @@ require (
 	github.com/hashicorp/waypoint-hzn v0.0.0-20201008221232-97cd4d9120b9
 	github.com/imdario/mergo v0.3.12
 	github.com/improbable-eng/grpc-web v0.13.0
-	github.com/kevinburke/go-bindata v3.22.0+incompatible
+	github.com/kevinburke/go-bindata v3.23.0+incompatible
```

Comment:
Is there a reason for this update?

Reply:
Whilst setting up the env for OSS Waypoint, I advised @cicoyle to run `go get github.com/kevinburke/go-bindata`, which resulted in the latest version being pulled. It looks like there were a few inconsequential updates between 3.22 and 3.23, so this could be rolled back if updating leads to issues.

Reply:
Interestingly, we don't appear to specify a version when pulling this for our release build https://github.com/hashicorp/waypoint-releases/blob/70d8ac07d4838b8b90d81bf5583ef5ad988e2648/.circleci/config.yml#L330